package.json

```diff
@@ -43,7 +43,7 @@
 		"format": "lerna run format --stream --parallel",
 		"test": "lerna run test --stream --parallel",
 		"test:unit": "lerna run test:unit --stream --parallel",
-		"test:integration": "lerna run test:integration --stream --parallel",
```

Comment:
What was the reason to remove `--parallel`?

Reply:
There were errors in in the integration tests for misuse of `nonce` of same account, because we are sharing same account in all tests. That's why had to move to run these tests in sequence.

Reply:
Hmm I wonder if there's something we can do it fix this, perhaps we could fetch the nonce before using it in the tests? I thought it's a big deficiency not to make use of `--parallel`

Reply:
Yes it is. The real solution would be to isolate the accounts for each tests. Means every test generate it's own accounts for testing. So it does not mixup with others.